### ISSUE TITLE
Description?

### ISSUE THREAD

**Issue by A2be** [MarsArtistsCommunity/ITS#1]
Is there a description for this one?

Is this a realistic-match model of what SpaceX put out? ... or a fanciful model inspired by what Space put out in September of this year?  ... or something else entirely?

(or maybe I should just ask, where is descriptive meta-text and meta-info kept for each of the models in this repo?)

**Comment by oceanbluesky** [comment]
Hi, a description of the ITS? at the top of the ITS repository in its description is a link to https://en.wikipedia.org/wiki/Interplanetary_Transport_System

This is a fairly realistic model in its current state compared to the official SpaceX renders, which are also only conceptual. As the model improves over time it should be by far the most professional high-resolution photoreal version of ITS...there is much to be improved upon: solar panels, texturing, engine nozzles, cockpit windows, cargo bay doors, elevator/crane system, particle system for thrusters/engine exhaust, interior views, on and on. 

"Are there descriptive meta-text and meta-info kept for each of the models in this repo?" Could you provide an example of what you are looking for? There should be a link above most models to wikipedia pages about the real-world system/vehicle/etc related to the models...

Cheers

**Comment by A2be** [comment]
My metadata question was just 'cause I thought that might be an easy way for you to answer.  In other words, if the file meta data of a file points to it being a realistic derivative work off of SpaceX-released renders, then that would answer the question on my mind.  Also, I'd like to figure out how the images here are licensed.  Are they Creative Commons so reuse is okay?

**Comment by octagonmars** [comment]
Yes all images and models are licensed for whatever use you wish. CC0

**Comment by A2be** [comment]
Thanks.  Good to know all are CCO.
